starknet_py/net/full_node_client.py

```diff
@@ -7,12 +7,13 @@
     Client,
 )
 from starknet_py.net.client_errors import ClientError
-from starknet_py.net.client_models import (
+from starknet_py.net.client_models import (  # pylint: disable=unused-import
     SentTransactionResponse,
     TransactionReceipt,
     BlockStateUpdate,
     StarknetBlock,
-    InvokeFunction,
+    InvokeFunction,  # backward compatibility
+    Invoke,
```

Comment:
I am not sure if we need to leave `InvokeFunction` imports in every place. If someone imported it from a wrong place they should adjust their import paths. What do you guys think?

Reply:
@cptartur @war-in

Reply:
I think it makes sense to do remove them.